tests/ui/implied-bounds/implied-bounds-on-nested-references-plus-variance.rs

```diff
@@ -1,15 +1,10 @@
-//@ check-pass
-//@ known-bug: #25860
```

Comment:
I guess I should probably rewrite this test to continue being a known-bug?

Reply:
Just make it higher-ranked in one more arg or sth...

Reply:
yes, and also add "regression test for X"